packages/core/package.json

```diff
@@ -17,7 +17,8 @@
   },
   "devDependencies": {
     "@medplum/definitions": "*",
-    "@medplum/fhirtypes": "*"
+    "@medplum/fhirtypes": "*",
+    "tinybench": "^2.5.0"
```

Comment:
I know @codyebberson tends to be nitpicky about the ^ on dependency versions if it could be removed